kasa/cli.py

```diff
@@ -323,12 +323,14 @@ async def emeter(dev: SmartDevice, year, month, erase):
 
     if year:
         click.echo(f"== For year {year.year} ==")
-        emeter_status = await dev.get_emeter_monthly(year.year)
+        click.echo(await dev.get_emeter_monthly(year.year))
+        return
```

Comment:
These returns will return from the function so no data will be printed out.

Reply:
I'm echoing the returned values and then exiting. I'm just not using emeter_status as intermediate variable. Data is absolutely printed.

Agreed that we can eliminate the returns if we adopt your change 2 above.

Reply:
I think there should be no intermediate status saving, considering the output format for realtime is different for the historical ones. If it looks like that the historical data formats are similar, it makes sense to share some of that code, though.